cci-config-generator.sh

```diff
@@ -15,7 +15,7 @@ remote_test_branch="$(git ls-remote git@"$vcs_provider".com:"${project}".git "$t
 local_test_branch="$(git branch -a | grep "$test_branch")"
 
 
-echo "git origin references \`${project}\`, which will be used as source."
+echo "git origin references \`${project}\` hosted by ${vcs_provider}, which will be used as source."
 
 # Read in API token
 read -rp 'Paste your CircleCI API token here: ' circle_token
```

Comment:
Nice work here!  Can you please move this line down to somewhere like line 71 so that users do not need to paste in their token until after we have determined they have a valid config.  This will save some user frustration.